music21/test/testStream.py

```diff
@@ -8009,6 +8009,15 @@ def testGetElementAfterElement(self):
     #     cLast = m10.notes[-1]
     #     self.assertEqual(cLast.expressions, [])
 
+    def testDottedTupletNotes(self):
```

Comment:
this test should be in duration.Duration not here, there is no need to create a Measure object to test it.

Reply:
I'll just note here that I'm a bit confused regarding the structure of the unit test classes and their locations. I'm used to projects with separate file/folder for unit tests.

I say that while saying that the 91% coverage now is already impressive.

Thank you for everything, hope that the feedback is taken in a good way

Reply:
I'll put this test in the currently right place anyway

Reply:
A lot of standard testing and documenting tools available today didn't exist when music21 was started, so this is why it does some things idiosyncratically -- but generally test cases called Test are what are run when testing the system.  TestExternal is where anything that has an external effect (playing sound, opening musescore, etc.) goes, and TestSlow is something that's too slow to test with every push but that needs to pass before a release.